microservices/realtime/pkg/utils/httputil.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+import "github.com/gin-gonic/gin"
+
+// NewError example
+func NewError(ctx *gin.Context, status int, err error) {
+	er := HTTPError{
+		Code:    status,
+		Message: err.Error(),
+	}
+	ctx.JSON(status, er)
+}
+
+// HTTPError example
+type HTTPError struct {
+	Code    int    `json:"code" example:"400"`
+	Message string `json:"message" example:"status bad request"`
+}
```

Comment:
Where is this used? 
Do you require HTTP response generators for anything?
Or gin-gonic?